pem/pubspec.yaml

```
@@ -13,10 +13,10 @@ topics:
  - parser
  - pem
 dependencies:
-  petitparser: ^5.1.0
+  petitparser: ^6.0.2
```

Any reason not to use a wider constraint?

```suggestion
  petitparser: '>=5.1.0 <7.0.0'
```